ONSdigital/eq-survey-runner
# Adding validation on number detail answers

### What is the context of this PR?
Added validation for the number of cars question. Making field a number field and making the max value able to be entered 20

This PR is dependant on the changes in this PR [#149](https://github.com/ONSdigital/eq-schema-validator/pull/149) and this PR [#586](https://github.com/ONSdigital/design-system/pull/586) (this will need to be merged first and then design system version upgraded before tests will pass)

### How to review 
- Enter non numeric values into field and check that it errors
- Enter a number of cars greater than 20 and check that it errors
